src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -100,11 +100,12 @@
   environment variables to be visible to the Ruby code. This map is by default
   set to {} if the user does not specify it in the configuration file."
   [env :- jruby-schemas/EnvMap
-   config :- (schema/pred map?)]
+   config :- jruby-schemas/JRubyConfig]
```

Comment:
@rlinehan do you call whether there was a reason we were using the looser schema here, rather than an explicit `JRubyConfig`?  I'd be fine with switching it back but it seemed like we could actually get away with being more strict here, and the tests pass, so... :)